src/test/java/org/ays/payload/Token.java

```diff
@@ -11,4 +14,20 @@ public class Token {
     private String refreshToken;
     private String accessTokenExpiresAt;
 
+    public static Token generateUserToken(UserCredentials userCredentials) {
+        Token token = new Token();
+        Response response = UserAuthEndpoints.getUserToken(userCredentials);
+        token.setAccessToken(response.jsonPath().getString("response.accessToken"));
+        token.setRefreshToken(response.jsonPath().getString("response.refreshToken"));
+        return token;
+    }
+
+    public static Token generateAdminToken(AdminCredentials adminCredentials) {
+        Token token = new Token();
+        Response response = InstitutionAuthEndpoints.getAdminToken(adminCredentials);
+        token.setAccessToken(response.jsonPath().getString("response.accessToken"));
+        token.setRefreshToken(response.jsonPath().getString("response.refreshToken"));
+        return token;
+
+    }
```

Comment:
Bu fonksiyonu buradan kaldırıp `InstitutionAuthEndpoints`'in içerisine taşıyabiliriz. Burası Token üretmiyor, Back-End servisine istek atarak Login oluyor aslında.